packages/core/config/webpack.common.js

```diff
@@ -191,6 +195,7 @@ module.exports = (webpackEnv) => {
           { from: 'public/', to: '', globOptions: { ignore: ['README.md'] } },
         ],
       }),
+      isEnvProduction && new CleanWebpackPlugin(),
```

Comment:
This makes sure the dist folder gets cleaned on each rebuild. We should have this anyway